js/map.js

```diff
@@ -320,11 +320,16 @@ function customMarkerGenerateJSON() {
     '     "dateAdded": "' + currentJSONDate + '"\n' +
     '   }\n</pre>';
 
-  document.getElementById('customMarkerJSONDiv').innerHTML = sJSON + '<input type="button" value="Start Over" onclick="clearWindows();" />';
+  document.getElementById('customMarkerJSONDiv').innerHTML = sJSON + '<input type="button" value="Submit Your Node" onclick="submitJson();"> <input style="margin-left:50px;" type="button" value="Start Over" onclick="clearWindows();" />';
 
 
 }
-
+function submitJson() {
+  var msg = 'I would like to add my node to the TOMesh node list.  I\'ve completed the JSON object below describing my node.\n\n```\n' + document.getElementById('jsonData').innerHTML + '```';
```

Comment:
Either `tomesh` all lower case, as that is how it appears everywhere, or spell out `Toronto Mesh`.